interpreter/while_test.go

```diff
@@ -24,14 +24,15 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/onflow/cadence/interpreter"
+	"github.com/onflow/cadence/test_utils"
 	. "github.com/onflow/cadence/test_utils/interpreter_utils"
 )
 
 func TestInterpretWhileStatement(t *testing.T) {
 
 	t.Parallel()
 
-	inter := parseCheckAndInterpret(t, `
+	invokable := test_utils.ParseCheckAndPrepare(t, `
```

Comment:
This is all we need to change in the test files: switch to the new API

Reply:
This is great! Maybe use a dot import for `test_utils` like we have for `interpreter_utils`

Reply:
done in ecdbe8e3a484ed1d2f9f8c5623d2fb17d8f198b5